client/web/compose/src/components/PageBlocks/ChartConfigurator.vue

```diff
@@ -119,12 +146,19 @@ export default {
       this.options.drillDown = {
         enabled: false,
```

Comment:
Instead of here, i suggest you have a method on the block itself, so you can just define it in one place and use it there